packages/code-du-travail-frontend/src/conventions/SearchModal.js

```diff
@@ -7,6 +7,16 @@ import ConventionSearch from "./Search";
 export const ConventionModal = ({ children: renderProp }) => {
   const [isModalVisible, setModalVisibility] = useState(false);
 
+  useEffect(() => {
+    document.body.addEventListener("tooltip-cc-event", () =>
+      setModalVisibility(true)
+    );
+    return () =>
+      document.body.removeEventListener("tooltip-cc-event", () =>
```

Comment:
to make that work you need to pass the fucntion you previously pass to addEventListener, so you will need create you function first.
what you have done here is create an anonymous function and pass it to addEventListener, create and anonymous function and pass it to removeEventListener...  
Rule of thumb #2: don't use anonymous function to add/remove event listener

Reply:
Bien vu !!! j'étais complètement passé au travers !

Reply:
OK effectivement